polkadot/node/subsystem-bench/src/lib/environment.rs

```diff
@@ -347,7 +347,7 @@ impl TestEnvironment {
 				break
 			}
 			// Check value every 50ms.
```

Comment:
```suggestion
			// Check value every 1000ms.
```

Reply:
But why?

Reply:
Accidentally changed it :D , will revert it back.